websites/G/Gemini/presence.ts

```diff
@@ -13,10 +13,27 @@ presence.on("UpdateData", async () => {
 			startTimestamp: browsingTimestamp,
 			details: "Browsing website",
 		},
-		{ pathname } = document.location;
+		{ pathname } = document.location,
+		showTitle = await presence.getSetting<boolean>("showTitle");
 	switch (true) {
 		case pathname.startsWith("/app"):
-			presenceData.details = "Asking questions";
+			// Check the selected conversation
+			if (showTitle) {
+				presenceData.details =
+					document.querySelector(
+						"div.conversation.selected>div.conversation-title"
+					).textContent ?? "Thinking of a new Prompt...";
+
+				// Show word count
+				const elem = document.querySelectorAll(
+						"p.query-text-line,div.response-optimization.markdown"
+					),
+					// eslint-disable-next-line no-one-time-vars/no-one-time-vars
+					askedWOrds = elem[0].textContent.split(" ").length,
+					// eslint-disable-next-line no-one-time-vars/no-one-time-vars
+					answerWords = elem[1].textContent.split(" ").length;
```

Comment:
Unless required, do not disable eslint rules

Reply:
okay resolved it